pkgs/os-specific/linux/power-profiles-daemon/default.nix

```diff
@@ -110,13 +112,6 @@ stdenv.mkDerivation rec {
     unset XDG_DATA_DIRS
   '';
 
-  postFixup = ''
-    # Avoid double wrapping
-    makeWrapperArgs+=("''${gappsWrapperArgs[@]}")
```

Comment:
Maybe we might not need `wrapGAppsNoGuiHook` and `dontWrapGApps` then. Looks like it does not depend on any GSettings schemas and the only needed typelibs are the default ones picked up by `pygobject3`.

Reply:
:+1: We don't need them.